docs/development.md

```diff
@@ -0,0 +1,126 @@
+# Developer Guide
+
+## Running examples locally
+
+The Hardware Event Proxy works with [Cloud Event Proxy](https://github.com/redhat-cne/cloud-event-proxy).
+Run cloud-event-proxy sidecar and consumer example from the cloud-event-proxy repo for testing locally.
+
+### Set environment variables
+```shell
+export NODE_NAME=mynode
+export HW_PLUGIN=true; export HW_EVENT_PORT=9087; export CONSUMER_TYPE=HW
+export MSG_PARSER_PORT=9097; export MSG_PARSER_TIMEOUT=10
+export LOG_LEVEL=debug
+# replace the following with real Redfish credentials and BMC ip address
+export REDFISH_USERNAME=admin; export REDFISH_PASSWORD=admin; export REDFISH_HOSTADDR=127.0.0.1
+
+```
+
+### Install and run Apache Qpid Dispach Router
+```shell
```

Comment:
you don't need AMQ Interconnect operator here?

Reply:
no. This is for local test, not in a cluster.